streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamSessionWindowAggregateProcessorTest.java

```diff
@@ -253,20 +300,29 @@ public void shouldRemoveMergedSessionsFromStateStore() {
 
     @Test
     public void shouldHandleMultipleSessionsAndMerging() {
+        time.sleep(1001L);
```

Comment:
Might it be easier to change the internal config of the "emit interval" and set it to zero (instead of advancing time)?

Reply:
Ack!